packages/frontend/src/features/sqlRunner/store/sqlRunnerSlice.ts

```diff
@@ -18,6 +22,39 @@ export const sqlRunnerSlice = createSlice({
         setProjectUuid: (state, action: PayloadAction<string>) => {
             state.projectUuid = action.payload;
         },
+        setInitialResultsTableConfig: (
```

Comment:
Why not have a default config in the store 'initialState'? Is this just so the initialisation pattern is always the same whether or not we are populating it from the database? 

I see that this is setting up all of the columns, but could the table render fine with a default config and not have entries for each column initially?

I don't have a strong opinion, just trying to understand.

Reply:
> I see that this is setting up all of the columns, but could the table render fine with a default config and not have entries for each column initially?

So since we rely on the `results` to render the columns config we need to set this after the query endpoint.

If we did set a default config in the store, then we would have to loop them somehow to render the table correctly 🤔 I'm not too sure on how this would look like but I think that at this moment we don't need to set a default because we still need to reference fields/columns